clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -153,6 +154,13 @@ public int memberEpoch() {
         return memberEpoch;
     }
 
+    @Override
+    public ConsumerGroupMetadata groupMetadata() {
+        // TODO: what do we use here, epoch?
```

Comment:
We should return the member epoch. The generationId name is a bit annoying though. I am not sure if we can do something about it.